### DISCUSSION TITLE
Be able to disable or relocate '/data/node-red-contrib-home-assistant-websocket.json'

### DISCUSSION THREAD

**Discussion by bjornstevens** [zachowj/node-red-contrib-home-assistant-websocket#507] [Feature Request]
Hi there,

In my case I use node-red as a Energy monitoring system for Home Assistant. I am a very happy user of this integration but I did notice something very annoying; the amount of writes.

While looking pretty much everywhere I found out that the 'node-red-contrib-home-assistant-websocket.json' file was doing these writes. In my situation I have around 8 values being updated nearly every second, causing the data also to be written to that file every half a second ( ~4 every 0.5 sec). 

Is there a way to relocate this file/ turn it off completely?

Thanks,
Bjorn

(do notice in the picture, it ain't the size of the file, it's just the amount of writes!)
![afbeelding](https://user-images.githubusercontent.com/23235234/140538153-61aebee2-3477-43b9-b045-2df0b88d52e7.png)

**Comment by zachowj** [comment]
I'm assuming you're using the entity node. It was saving the payload on each input even if resend wasn't checked. I changed it to only save when resend is checked. So if that was what was causing all your disk writes it was changed in the latest release.

**Reply by bjornstevens** [reply]
Hi, Thanks for the reply!

What does the resend mean/do?

**Reply by zachowj** [reply]
<https://zachowj.github.io/node-red-contrib-home-assistant-websocket/node/entity.html#resend-state-and-attributes>

**Comment by bjornstevens** [comment]
Thanks for the fix in the relase @zachowj ! Works great now! 💯

**Comment by bjornstevens** [comment]
Btw, also a funny backstory:

My SSD had an insane amount of writes, here is the difference between the disks where node-red is on VS a SSD with VM's:
![afbeelding](https://user-images.githubusercontent.com/23235234/140618626-ff39f3f4-941e-432e-8bf8-53912fafaa27.png)

Node-red had a lot of writes comming in per second. In total I would guess around 10 per second. In total that would make it 864.000 writes per day 😆. This fix litterally saves my SSD's lives lol.
